rllib/models/tf/recurrent_net.py

```diff
@@ -8,14 +8,14 @@
 
 
 @DeveloperAPI
-class RecurrentTFModelV2(TFModelV2):
+class RecurrentNetwork(TFModelV2):
```

Comment:
This change seems fine, but note that it is a breaking DeveloperAPI change.

Reply:
+1, we could do this in a separate PR, maybe and/or just make sure, the old names still work (and cause a warning). I wanted to clean this up, because none of these names fit into the same pattern. And then attention was yet another new net type.